src/js/components/HeaderBar.tsx

```diff
@@ -18,6 +21,19 @@ export default class HeaderBar extends React.Component {
           </Link>
         </div>
         <div className="header-bar-right-align-wrapper">
+          {refreshRate > 2 ? (
```

Comment:
if we want to show that if we're deviating from the default, why don't we express that here?

something like `currentRefreshRate != defaultRefreshRate`?